subsys/bluetooth/host/hci_ecc.c

```diff
@@ -369,6 +372,9 @@ static void le_gen_dhkey_v1(struct net_buf *buf)
 	struct bt_hci_cp_le_generate_dhkey *cmd;
 	uint8_t status;
 
+	net_buf_pull_u8(buf);
+	net_buf_pull(buf, sizeof(struct bt_hci_cmd_hdr));
```

Comment:
Should we have an ASSERT on the type here?

Reply:
I'll add it :+1:. It's just awkward with the unused variable warning.

Reply:
Ah, right. And the assert macro that supports this is somewhat even more awkward to use